freediscovery/server/resources.py

```diff
@@ -846,3 +848,31 @@ def post(self, **args):
         res = model.fe.db.render_dict(scores_pd)
 
         return {'data': res}
+
+
+# ============================================================================ #
+#                            Custom Stop Words
+# ============================================================================ #
+
+
+class CustomStopWordsApi(Resource):
+    @doc(description="Perform a mechanism for adding / managing custom stop words")
+    @use_args({"name" : wfields.Str(required=True),
+               "stop_words" : wfields.List(wfields.Str(), required=True)})
+    @marshal_with(CustomStopWordsSchema())
+    def post(self, **args):
+        self.name = args['name']
+        self.stop_words = args['stop_words']
+        self.model = _StopWordsWrapper(cache_dir=self._cache_dir)
+        self.model.save(name = self.name, stop_words = self.stop_words)
+        return self.model
+
+class CustomStopWordsLoadApi(Resource):
+    @doc(description="Loading a custom stop words")
+    def get(self, **args):
+        name = args['name']
+        from sklearn.externals.joblib import load
+        model_dir = os.path.join(self._cache_dir, 'stop_words')
+        n = os.path.join(model_dir, name + '.pkl')
```

Comment:
This should use a `_StopWordsWrapper.load` method; the idea is that all the actual calculations / IO should be done in that class and `CustomStopWordsLoadApi` is just thin wrapper for input validation / output formatting